packages/cli/generators/extension/templates/index.js

```diff
@@ -1,5 +1,5 @@
-// Copyright IBM Corp. 2017. All Rights Reserved.
-// Node module: <%= project.name %>
+// Copyright IBM Corp. 2017,2018. All Rights Reserved.
+// Node module: @loopback/cli
```

Comment:
qq: do we put the our package name here or the project's name?

Reply:
Good catch, and I don't know.

@rmg do you have any opinion on what license header (if any) should be placed in the source code files emitted by CLI tool for scaffolding new projects? Since the content of the file will eventually be owned by the user of our CLI tool, I think it would be best to leave the license header out.

Thoughts?

Reply:
The template itself should probably have a copyright notice.. but what it produces shouldn't have IBM's name in it I think. One way around this is to change the file extension to `.ejs` or something since the copyright tool is specifically looking for `.js` files.

Reply:
The templates _should_ have a copyright header in them, that's true. But it looked to me like the lines that were replaced were part of the template itself, meant to be replaced by the user's copyright notice.